packages/shared/src/components/comments/CommentAuthor.tsx

```diff
@@ -2,21 +2,21 @@ import React, { ReactElement } from 'react';
 import classNames from 'classnames';
 import { Author } from '../../graphql/comments';
 import { ProfileLink } from '../profile/ProfileLink';
-import FeatherIcon from '../icons/Feather';
 import { ProfileTooltip } from '../profile/ProfileTooltip';
+import { UserBadgeProps } from '../UserBadge';
 
 export interface CommentAuthorProps {
-  postAuthorId: string | null;
   author: Author;
   className?: string;
   appendTooltipTo?: () => HTMLElement;
+  badges?: ReactElement<UserBadgeProps>[];
```

Comment:
The squadMemberRoleBadge is not a UserBadgeProps right?
Not sure if that's correct now

Reply:
I guess our typescript does not work 😆